force-app/main/default/classes/RP_GitHubController.cls

```diff
@@ -34,100 +34,10 @@
 * @description Component that brings the latest release notes from Github using Salesforce org HEROKU APP.
 */
 public with sharing class RP_GitHubController {
-
-    //Object used to load all information related to github release notes
-    public GitHubWrapper gitHubData {get; set;}
-    public Boolean timeoutError {get; set;}
-    public Boolean remoteSiteSettingIsNotActive {get; set;}
+    // This class is slated for future deprecation and removal.
```

Comment:
I noticed that you stripped down some classes as marked as "to be deleted", such as this one. But in other cases the classes are just deleted (`RP_GitHubClient`, `RP_GitHubTestMock`, etc.). Is there a reason some are deleted and others are not?